lib/util/src/hse_err.c

```diff
@@ -24,11 +25,13 @@ extern uint8_t __start_hse_merr;
 extern uint8_t __stop_hse_merr;
 
 merr_t
-merr_pack(int errnum, const char *file, int line)
+merr_pack(int errnum, const uint16_t ctx, const char *file, int line)
 {
     merr_t err = 0;
     s64    off;
 
+    INVARIANT(errnum >= INT16_MIN && errnum <= INT16_MAX);
```

Comment:
Rename local var `errrnum` to `errno_value` to make it clear it is expected to be an errno value?

Reply:
maybe INVARIANT(errnum >= 0 && ctx >= 0 && ctx < HSE_ERR_CTX_MAX && file && line > 0);